js/modules/k6/http/request_test.go

```diff
@@ -83,17 +80,16 @@ func assertRequestMetricsEmitted(t *testing.T, sampleContainers []metrics.Sample
 				assert.Equal(t, strconv.Itoa(status), tags["status"])
 				assert.Equal(t, method, tags["method"])
 				assert.Equal(t, group, tags["group"])
-				assert.Equal(t, name, tags["name"])
 			}
 		}
 	}
-	assert.True(t, seenDuration, "url %s didn't emit Duration", url)
-	assert.True(t, seenBlocked, "url %s didn't emit Blocked", url)
-	assert.True(t, seenConnecting, "url %s didn't emit Connecting", url)
-	assert.True(t, seenTLSHandshaking, "url %s didn't emit TLSHandshaking", url)
-	assert.True(t, seenSending, "url %s didn't emit Sending", url)
-	assert.True(t, seenWaiting, "url %s didn't emit Waiting", url)
-	assert.True(t, seenReceiving, "url %s didn't emit Receiving", url)
+	assert.True(t, seenDuration, "url %s didn't emit Duration", nameAndURL)
+	assert.True(t, seenBlocked, "url %s didn't emit Blocked", nameAndURL)
+	assert.True(t, seenConnecting, "url %s didn't emit Connecting", nameAndURL)
+	assert.True(t, seenTLSHandshaking, "url %s didn't emit TLSHandshaking", nameAndURL)
+	assert.True(t, seenSending, "url %s didn't emit Sending", nameAndURL)
+	assert.True(t, seenWaiting, "url %s didn't emit Waiting", nameAndURL)
+	assert.True(t, seenReceiving, "url %s didn't emit Receiving", nameAndURL)
```

Comment:
Looking for the changes, it seems like in most places, it doesn't really matter that variable names `nameAndURL` moreover, for the checks like above it even confuses :thinking: 

Maybe we stick with the just `url` variable name and comment on the assertion 

```
// URL equals name, because https://github.com/grafana/k6/issues/2584#issuecomment-1219618111
assert.Equal(t, tags["url"], tags["name"])
```

Or is any better shorter explanation, WDYT?

Reply:
I addressed it in the latest force-pushed commit, let me know